client-sdk/ts-web/core/test/hdkey.test.ts

```diff
@@ -0,0 +1,71 @@
+import {HDKey} from '../src/hdkey';
+
+interface TestVector {
+    seed: string;
+    tests: {
+        path?: string;
+        chainCode: string;
+        private: string;
+        public: string;
+    }[];
+}
+
+interface Adr0008Vector {
+    kind: string,
+    bip39_mnemonic: string,
+    bip_passphrase: string,
+    bip39_seed: string,
+    oasis_accounts: {
+        private_key: string,
+        public_key: string,
+        address: string
+    }[]
+}
+
+const adr0008Vectors: Adr0008Vector[] = require('./adr-0008-vectors.json');
+const slip10Vectors: TestVector[] = require('./slip-0010-vectors.json');
+
+const uint2hex = (array: Uint8Array) => Buffer.from(array).toString('hex');
+const uint2base64 = (array: Uint8Array) => Buffer.from(array).toString('base64');
+
+describe('HDKey', () => {
+    describe('Adr-0008 Vectors', () => {
+        // @Todo waiting on oasis-core test vectors
+        adr0008Vectors.forEach((vector, index) => {
+            it(`Case #${index}`, async () => {
+                // This can be a bit slow on CI servers.
+                jest.setTimeout(10000)
```

Comment:
where did your semis go?

Reply:
Oops, I'm strongly anti-semi so I usually let the linter add them in for me - I added the test folder to the lint/fmt commands so that they're also linted.